.github/workflows/e2e.yml

```diff
@@ -77,18 +102,28 @@ jobs:
           synapse get -r --downloadLocation "${{ env.LOCAL_TEAM_IMAGES_DIR }}/" "${{ vars.TEAM_IMAGES_ID }}"
 
       - name: build image
+        if: ${{ env.IMAGE_EXISTS == 'false'}}
         env:
           DOCKERFILE_PATH: './docker/Dockerfile'
         run: |
           docker build \
-          -t "${{ env.DATA_IMAGE_TAG }}" \
+          -t "${{ env.DATA_IMAGE_PATH }}" \
           -f "${{ env.DOCKERFILE_PATH }}" \
+          --build-arg SOURCE_REPO="https://github.com/${{ env.NAMESPACE }}/${{ vars.DB_NAME }}" \
+          --build-arg DESCRIPTION="${GITHUB_SHA::7}" \
           --build-arg LOCAL_DATA_PATH="${{ env.LOCAL_DATA_PATH }}" \
           --build-arg LOCAL_COLLECTIONS_PATH="${{ env.LOCAL_COLLECTIONS_PATH }}" \
           --build-arg LOCAL_IMPORT_SCRIPT_PATH="${{ env.LOCAL_IMPORT_SCRIPT_PATH }}" \
           --build-arg LOCAL_CREATE_INDEXES_PATH="${{ env.LOCAL_CREATE_INDEXES_PATH }}" \
           .
-      # TODO - push image to remote repository
+
+      # Note: if a new package is created using this workflow, then the package
+      # visibility will default to the source repository visbility. For example,
```

Comment:
typo:visibility

Reply:
Thanks for the catch! I'll update in the next commit.